deflect/NetworkListener.h

```diff
@@ -82,12 +83,11 @@ public slots:
     void onEventRegistrationReply( QString uri, bool success );
 
 private:
+    detail::NetworkListener* _impl;
```

Comment:
I am totally open to using leading _ for private members, but the convention in this project is to put them as a prefix. Do we want to gradually change it everywhere, or do we respect the existing practice..?